packages/variable-resolver/src/browser/variable-resolver-frontend-contribution.ts

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (C) 2018 Red Hat, Inc.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+
+import { injectable, inject, named } from 'inversify';
+import { FrontendApplicationContribution } from '@theia/core/lib/browser';
+import { Command, CommandContribution, CommandRegistry, ContributionProvider } from '@theia/core/lib/common';
+import { VariableContribution, VariableRegistry } from './variable';
+import { VariableQuickOpenService } from './variable-quick-open-service';
+
+export const LIST_VARIABLES: Command = {
+    id: 'variable.list',
+    label: 'Variable: list all'
```

Comment:
Could we have `Variable: List All` instead? At least, that was the convention for instance in the Git extension so far.

Reply:
Updated